client/web/app/elements/lecture/editor.html

```diff
@@ -192,6 +192,9 @@
                     return;
                 }
 
+                if (ArrayUtils.isNotEmpty(this.lecture.mappings)) {
+                    return;
```

Comment:
W sytuacji gdy zmienimy link fimu albo prezentacji, to musimy generowac nowe mapowania, to nam zablokuje. DO zmiany w przyszlosci